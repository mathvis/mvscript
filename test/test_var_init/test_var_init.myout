Decl (Variable (VarIdentifier "a") (Just IntT) (Just (Type (Int 0))))
Decl (Variable (VarIdentifier "a") (Just IntT) (Just (Type (Int 0))))
--> test/test_var_init/test_var_init.mvs:3:1
error: expected StringT but got IntT
