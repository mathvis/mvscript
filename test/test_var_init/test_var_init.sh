#!/bin/bash
helix test_var_init.mvs test_var_init.out --vsplit
