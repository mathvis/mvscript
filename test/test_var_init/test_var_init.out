Decl (Variable (VarIdentifier "a") (Just IntT) (Just (Type (Int 0))))
Decl (Variable (VarIdentifier "a") (Just IntT) (Just (Type (Int 0))))
error: expected StringT but got IntT
 --> test/test_var_init/test_var_init.mvs:3:1
  |
3 | let a: string = 0;
  | ^
  |
hint: Consider changing the variable type or declaring a new variable.
